apps/boilerplate-front-end/src/components/facets/components/CurrentRefinement.jsx

```diff
@@ -109,10 +70,23 @@ function CurrentRefinementGeneral(props) {
             <a
               onClick={(event) => {
                 event.preventDefault()
+
+                if (
```

Comment:
What is this for?

Reply:
Navigation state back to null when removing the custom filter (accissible through the navigation bar Woman accessories even & odd)

Reply:
Pushed some comment for clearer explainations